src/epub/text/chapter-5.xhtml

```diff
@@ -57,20 +57,19 @@
+				<p>“Ladder nothing! The boss is right. See here, <abbr epub:type="z3998:name-title">Mr.</abbr> Headley, you put in that letter that you are going to send the universe⁠—my! don’t I see the scare lines in the journals!⁠—that we are at 27 North Latitude and 28.14 West Longitude, or whatever other figure is the right one. Got that? Then you say that three of the most important folk in history, the great man of Science, Maracot, and the rising-star bug-collector, Headley, and Bob Scanlan, a peach of a mechanic and the pride of Merribank’s; are all yellin’ and whoopin’ for help from the bottom of the sea. Follow my idea?”</p>
 				<p>“Well, what then?”</p>
 				<p>“Well, then it’s up to them, you see. It’s kind of a challenge that they can’t forget. Same as I’ve read of Stanley finding Livingstone and the like. It’s for them to find some way to yank us out or to catch us at the other end if we can take the jump ourselves.”</p>
 				<p>“We could suggest the way ourselves,” said the Professor. “Let them drop a deep-sea line into these waters and we will look out for it. When it comes we can tie a message to it and bid them stand by for us.”</p>
 				<p>“You’ve said a mouthful!” cried Bob Scanlan. “That is sure the way to do it.”</p>
 				<p>“And if any lady cared to share our fortunes four would be as easy as three,” said Maracot, with a roguish smile at me.</p>
 				<p>“For that matter, five is as easy as four,” said Scanlan. “But you’ve got it now, <abbr epub:type="z3998:name-title">Mr.</abbr> Headley. You write that down, and in six months we shall be back in London River once more.”</p>
-				<p>So now we launch our two balls into that water which is to us what the air is to you. Our two little balloons will go aloft. Will both be lost on the way? It is possible. Or may we hope that one will get through? We leave it on the knees of the gods. If nothing can be done for us, then let those who care for us know that in any case we are safe and happy. If, on the other hand, this suggestion could be carried out and the money and energy for our rescue should be forthcoming, we have given you the means by which it can be done. Meanwhile, goodbye⁠—or is it au revoir?</p>
+				<p>So now we launch our two balls into that water which is to us what the air is to you. Our two little balloons will go aloft. Will both be lost on the way? It is possible. Or may we hope that one will get through? We leave it on the knees of the gods. If nothing can be done for us, then let those who care for us know that in any case we are safe and happy. If, on the other hand, this suggestion could be carried out and the money and energy for our rescue should be forthcoming, we have given you the means by which it can be done. Meanwhile, goodbye⁠—or is it <i xml:lang="fr">au revoir</i>?</p>
```

Comment:
This should also be without italics, as au revoir is in M-W: https://www.merriam-webster.com/dictionary/au%20revoir

I thought I had noted this in my earlier review but I guess I forgot to submit the comment, sorry!

Reply:
I'll pop out confrères, too, then: https://www.merriam-webster.com/dictionary/confreres

I wasn't familiar with that one so I jumped to conclusions, but I guess I was just [one of that day's 10,000](https://xkcd.com/1053/).